src/uct/ib/dc/dc_mlx5_ep.c

```diff
@@ -1522,6 +1541,12 @@ ucs_status_t uct_dc_mlx5_ep_check_fc(uct_dc_mlx5_iface_t *iface,
         }
 
         goto out;
+    } else {
```

Comment:
Do we really need else here, given the goto ?

Reply:
removed